packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -441,19 +481,33 @@ export class LocalIntervalCollection<TInterval extends ISerializableInterval> {
         start: number,
         end: number,
         intervalType: MergeTree.IntervalType,
-        props?: MergeTree.PropertySet) {
-        const interval = this.createInterval(start, end, intervalType);
+        props: MergeTree.PropertySet,
```

Comment:
>props [](http://example.com/codeflow?start=8&length=5)

why make this not optional?

Reply:
ahh this is the local collection, so it's internal only, hard to keep all the classes straight in this file